### PR TITLE
remove deprecated Styles in the ASM module

### PR DESCRIPTION
**Pull request by niehuayang**
JIRA: https://jira.tools.sap/browse/CXSPA-2410

### CODE REVIEW

File: feature-libs/asm/styles/components/_customer-selection.component.scss [hunk 1]
```diff
@@ -63,27 +57,23 @@
     }
 
     button[type='submit'] {
-      @include forVersion(1, 5) {
```
Comment (ercultimate) [inline feature-libs/asm/styles/components/_customer-selection.component.scss:66]: It looks like this block was kept, when it should have been dropped.  Could you please re-verify this change?

**Review by ercultimate — CHANGES_REQUESTED** [review]

File: feature-libs/asm/styles/components/_asm-main-ui.component.scss [hunk 2]
```diff
@@ -134,12 +124,10 @@
...
     }
 
-    @include forVersion(5.1) {
-      color: inherit;
-      background-color: #ffffff;
-      box-shadow: 2px 2px rgba(85, 107, 130, 0.1),
-        inset 0px -1px 0px rgba(85, 107, 130, 0.2);
-    }
+    color: inherit;
+    background-color: #ffffff;
```
Comment (morganm58) [inline feature-libs/asm/styles/components/_asm-main-ui.component.scss:128]: Is there a reason why asm doesn't use the color variables?

File: feature-libs/asm/styles/components/_customer-selection.component.scss (hunk 1 above)
Comment (niehuayang) [reply feature-libs/asm/styles/components/_customer-selection.component.scss:66]: Ah, you are right,  I am careless. now I have corrected it

File: feature-libs/asm/styles/components/_asm-main-ui.component.scss (hunk 2 above)
Comment (niehuayang) [reply feature-libs/asm/styles/components/_asm-main-ui.component.scss:128]: Hi @ercultimate, Do you know this reason?

**Review by morganm58 — DISMISSED** [review]

File: feature-libs/asm/styles/components/_asm-main-ui.component.scss (hunk 2 above)
Comment (ercultimate) [reply feature-libs/asm/styles/components/_asm-main-ui.component.scss:128]: ASM components use a separate UI theme from the Spartacus storefront, so it should not be using the storefront theme variables.

**Review by ercultimate — CHANGES_REQUESTED** [review]
The changes are good.  We just have to add the CSS changes to the styles migration document to meet the DoD.

**Comment by niehuayang** [comment]
> The changes are good. We just have to add the CSS changes to the styles migration document to meet the DoD.

Hi Eric, I have add the CSS changes to the styles migration document, could you please help me review it?

File: docs/migration/6_0/styling.md [hunk 3]
```diff
@@ -69,7 +69,41 @@
...
- 
+
+### Feature lib asm 
+
+#### AsmMainUiComponent
+
+- Added property `background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='black' d='M14.53 4.53l-1.06-1.06L9 7.94 4.53 3.47 3.47 4.53 7.94 9l-4.47 4.47 1.06 1.06L9 10.06l4.47 4.47 1.06-1.06L10.06 9z'/%3E%3C/svg%3E");` to `.close`
+- Added property `background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' viewBox='0 0 16 16'%3E%3Cpath fill='%230064D9' d='M11,2.7c1.2,0.6,2.2,1.5,2.9,2.6c1.3,1.9,1.5,4.4,0.6,6.5c-0.3,0.8-0.8,1.6-1.5,2.2c-0.6,0.6-1.4,1.1-2.2,1.5 C9.9,15.8,9,16,8,16c-0.9,0-1.9-0.2-2.7-0.5c-0.8-0.4-1.6-0.9-2.2-1.5c-0.6-0.6-1.1-1.4-1.5-2.2C0.7,9.6,0.9,7.2,2.1,5.3 c0.7-1.1,1.7-2,2.9-2.6v1.1C4.1,4.3,3.3,5.1,2.8,6C2.3,6.9,2,7.9,2,9c0,1.6,0.6,3.2,1.8,4.3c0.5,0.5,1.2,1,1.9,1.3 c1.5,0.6,3.2,0.6,4.7,0c0.7-0.3,1.4-0.7,1.9-1.3C13.4,12.1,14,10.6,14,9c0-1.1-0.3-2.1-0.8-3c-0.5-0.9-1.3-1.7-2.2-2.2 C11,3.8,11,2.7,11,2.7z M8,9C7.7,9,7.5,8.9,7.3,8.7C7.1,8.5,7,8.3,7,8V1c0-0.3,0.1-0.5,0.3-0.7c0.4-0.4,1-0.4,1.4,0 C8.9,0.5,9,0.7,9,1v7c0,0.3-0.1,0.5-0.3,0.7C8.5,8.9,8.2,9,8,9z'/%3E%3C/svg%3E%0A");` to `.logout`
```
Comment (ercultimate) [inline docs/migration/6_0/styling.md]: Oh, this really made me laugh. 🤣 
I'm going to ask if we could abbreviate URIs.

**Review by ercultimate — APPROVED** [review]
I think this is acceptable for now.  I'll find out about the URIs and let you know.

File: docs/migration/6_0/styling.md (hunk 3 above)
Comment (niehuayang) [reply docs/migration/6_0/styling.md]: this is a good question! That's what I want to ask

Comment (ercultimate) [inline docs/migration/6_0/styling.md]:
```suggestion
- Added property `background-image: url("[..IMAGE_URI...]");` to `.logout`
```

File: docs/migration/6_0/styling.md [hunk 4]
```diff
@@ -69,7 +69,41 @@
...
+- Added property `color: inherit; background-color: #ffffff; box-shadow: 2px 2px rgba(85, 107, 130, 0.1), inset 0px -1px 0px rgba(85, 107, 130, 0.2);` to `.asm-bar`
+
+#### AsmSessionTimerComponent
+
+- Added property `background: url("data:image/svg+xml,%0A%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' viewBox='0 0 16 16'%3E%3Cpath fill='%230064D9' d='M14.9,7.5l-1,0.2c0.2,0.9,0.1,1.7-0.1,2.5c-0.3,1-0.8,2-1.5,2.7c-1.1,1.1-2.7,1.8-4.2,1.8 c-0.8,0-1.5-0.1-2.3-0.4c-1.5-0.6-2.7-1.8-3.3-3.3C2.1,10.2,2,9.5,2,8.7c0-1.6,0.7-3.1,1.8-4.3c0.7-0.8,1.7-1.3,2.7-1.5 c1-0.3,2-0.2,3,0l0,0v-1c-1-0.2-2.1-0.2-3.1,0C4.2,2.4,2.4,4,1.5,6.1C1.2,6.9,1,7.8,1,8.7c0,0.9,0.2,1.8,0.5,2.6 c0.4,0.9,0.9,1.7,1.5,2.3c0.7,0.7,1.4,1.2,2.3,1.5c0.8,0.3,1.7,0.5,2.6,0.5c0.9,0,1.8-0.2,2.6-0.5c2.1-0.9,3.7-2.7,4.2-5 C15,9.3,15,8.4,14.9,7.5z'/%3E%3Cpolygon fill='%23d1e3ff' points='11.5,2.8 9.2,4.5 9.7,0.5 '/%3E%3C/svg%3E%0A") no-repeat center center;` to `.reset`
```
Comment (ercultimate) [inline docs/migration/6_0/styling.md]:
```suggestion
- Added property `background: url("[...IMAGE_URI...]") no-repeat center center;` to `.reset`
```

File: docs/migration/6_0/styling.md [hunk 5]
```diff
@@ -69,7 +69,41 @@
...
+
+#### AsmToggleUiComponent
+
+- Added property `color: #d50101;` to `.label`
+- Added property `background: url("data:image/svg+xml,%3Csvg aria-hidden='true' focusable='false' data-prefix='fas' data-icon='chevron-circle-up' class='svg-inline--fa fa-chevron-circle-up fa-w-16' role='img' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 512 512'%3E%3Cpath fill='%230064D9' d='M8 256C8 119 119 8 256 8s248 111 248 248-111 248-248 248S8 393 8 256zm231-113.9L103.5 277.6c-9.4 9.4-9.4 24.6 0 33.9l17 17c9.4 9.4 24.6 9.4 33.9 0L256 226.9l101.6 101.6c9.4 9.4 24.6 9.4 33.9 0l17-17c9.4-9.4 9.4-24.6 0-33.9L273 142.1c-9.4-9.4-24.6-9.4-34 0z'%3E%3C/path%3E%3C/svg%3E") center center no-repeat;` to `.collapseIcon`
```
Comment (ercultimate) [inline docs/migration/6_0/styling.md]:
```suggestion
- Added property `background: url("[...IMAGE_URI...]") center center no-repeat;` to `.collapseIcon`
```

File: docs/migration/6_0/styling.md [hunk 6]
```diff
@@ -69,7 +69,41 @@
...
+- Added property `background: url("data:image/svg+xml,%3Csvg aria-hidden='true' focusable='false' data-prefix='fas' data-icon='chevron-circle-down' class='svg-inline--fa fa-chevron-circle-down fa-w-16' role='img' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 512 512'%3E%3Cpath fill='%230064D9' d='M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z'%3E%3C/path%3E%3C/svg%3E") center center no-repeat;` to `.expandIcon`
```
Comment (ercultimate) [inline docs/migration/6_0/styling.md]:
```suggestion
- Added property `background: url("[...IMAGE_URI...]") center center no-repeat;` to `.expandIcon`
```

**Review by ercultimate — DISMISSED** [review]
Here is a way to abbreviate the URIs for better readability.

File: docs/migration/6_0/styling.md (hunk 3 above)
Comment (ercultimate) [reply docs/migration/6_0/styling.md]: Yeah, they agree this is a bit much.  
I suggest replacing URIs with: `[...IMAGE_URI...]` 
If a value is being changed, use: `[...OLD_IMAGE_URI...]` and `[...NEW_IMAGE_URI...]`.

Comment (niehuayang) [reply docs/migration/6_0/styling.md]: Cool, this a smart way

Comment (niehuayang) [reply docs/migration/6_0/styling.md]: changed

File: docs/migration/6_0/styling.md (hunk 4 above)
Comment (niehuayang) [reply docs/migration/6_0/styling.md]: changed

File: docs/migration/6_0/styling.md (hunk 5 above)
Comment (niehuayang) [reply docs/migration/6_0/styling.md]: changed

File: docs/migration/6_0/styling.md (hunk 6 above)
Comment (niehuayang) [reply docs/migration/6_0/styling.md]: changed

**Review by kimhw0630 — APPROVED** [review]

**Pull request merged**